core-bundle/src/Resources/contao/library/Contao/System.php

```diff
@@ -614,70 +592,6 @@ public static function isInstalledLanguage($strLanguage)
 		return static::$arrLanguages[$strLanguage];
 	}
 
-	/**
-	 * Return the countries as array
-	 *
-	 * @return array An array of country names
-	 *
-	 * @deprecated Deprecated since Contao 4.12, to be removed in Contao 5;
-	 *             use the Contao\CoreBundle\Intl\Countries service instead
-	 */
-	public static function getCountries()
-	{
-		trigger_deprecation('contao/core-bundle', '4.12', 'Using the %s method has been deprecated and will no longer work in Contao 5.0. Use the "contao.intl.countries" service instead.', __METHOD__);
-
-		$arrCountries = self::getContainer()->get('contao.intl.countries')->getCountries();
-
-		return array_combine(array_map('strtolower', array_keys($arrCountries)), $arrCountries);
-	}
-
-	/**
-	 * Return the available languages as array
-	 *
-	 * @param boolean $blnInstalledOnly If true, return only installed languages
-	 *
-	 * @return array An array of languages
-	 *
-	 * @deprecated Deprecated since Contao 4.12, to be removed in Contao 5;
-	 *             use the Contao\CoreBundle\Intl\Locales service instead
-	 */
-	public static function getLanguages($blnInstalledOnly=false)
-	{
-		trigger_deprecation('contao/core-bundle', '4.12', 'Using the %s method has been deprecated and will no longer work in Contao 5.0. Use the "contao.intl.locales" service instead.', __METHOD__);
-
-		if ($blnInstalledOnly)
-		{
-			return self::getContainer()->get('contao.intl.locales')->getEnabledLocales(null, true);
-		}
-
-		return self::getContainer()->get('contao.intl.locales')->getLocales(null, true);
-	}
-
-	/**
-	 * Return the timezones as array
-	 *
-	 * @return array An array of timezones
-	 */
-	public static function getTimeZones()
-	{
-		trigger_deprecation('contao/core-bundle', '4.13', 'Using the %s method has been deprecated and will no longer work in Contao 5.0. Use the DateTimeZone::listIdentifiers() instead.', __METHOD__);
-
-		$arrReturn = array();
-		$timezones = array();
-
-		require __DIR__ . '/../../config/timezones.php';
```

Comment:
The file can die too, right?

Reply:
Indeed. :slightly_smiling_face: 

Removed in 9213fb9c73363a1ec7ddccbb3d81d2994a951dd1